src/objective-c/GRPCClient/GRPCCall.h

```diff
@@ -155,6 +155,19 @@ typedef NS_ENUM(NSUInteger, GRPCErrorCode) {
 };
 
 /**
+ * Flags for options of a gRPC call
+ *
+ */
+typedef NS_OPTIONS(NSUInteger, GRPCCallFlags) {
+  /** Signal that the call is idempotent */
+  GRPCFlagIdempotentRequest = 0x00000010,
+  /** Signal that the call should not return UNAVAILABLE before it has started */
+  GRPCFlagIgnoreConnectivity = 0x00000020,
+  /** Signal that the call is cacheable. GRPC is free to use GET verb */
+  GRPCFlagCacheableRequest = 0x00000040,
```

Comment:
I would prefer to use mask considering extensibility; there might be other flags in the future that are orthogonal to these two flags. I would put comment for "cacheable" that when it is set "idempotent" is automatically considered set. How do you think?